client/js/utils/dom-events.js

```diff
@@ -5,3 +5,8 @@ import throttle from 'xstream/extra/throttle';
 export const windowResize$ = fromEvent(window, 'resize').compose(debounce(500));
 export const windowScroll$ = fromEvent(window, 'scroll').compose(throttle(100));
 export const windowOrientationChange$ = fromEvent(window, 'orientationchange');
+export const documentReadyState$ =
```

Comment:
I've change it to be `onEventName$` when it's an event, and `valueName$` when it's a value that changes over time... How's that?